test/library/helpers.go

```diff
@@ -0,0 +1,124 @@
+package library
+
+import (
+	"context"
+
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	routeclient "github.com/openshift/client-go/route/clientset/versioned"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+	watchtools "k8s.io/client-go/tools/watch"
+	"k8s.io/klog"
+)
+
+// ClientConfig returns a config configured to connect to the api server
+func GetClientConfig() *rest.Config {
+	loader := clientcmd.NewDefaultClientConfigLoadingRules()
+	clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		loader,
+		&clientcmd.ConfigOverrides{
+			ClusterInfo: api.Cluster{
+				InsecureSkipTLSVerify: true,
+			},
+		},
+	)
+
+	config, err := clientConfig.ClientConfig()
+	if err != nil {
+		panic(err)
+	}
+
+	return config
+}
+
+func KubeClient() *kubernetes.Clientset {
+	kubeClient, err := kubernetes.NewForConfig(GetClientConfig())
+	if err != nil {
+		panic(err)
+	}
+
+	return kubeClient
+}
+
+func RouteClient() *routeclient.Clientset {
+	routeClient, err := routeclient.NewForConfig(GetClientConfig())
+	if err != nil {
+		panic(err)
+	}
+
+	return routeClient
+}
+
+func ConfigClient() *configclient.Clientset {
```

Comment:
I'd prefer we don't introduce these above, they are simple enough not be helpers.

Reply:
For me, the advantage is they avoid the error handling which differs when run as test only. It could also be done with `runtime.Must` pattern but I made it this way to follow the pattern we have in regular e2e's. I don't feel strong though - It's simplicity + consistency vs. deps which don't matter that much in test packages.